### ISSUE TITLE
[bug] White space below taskbar

### ISSUE THREAD

**Issue by J4gQBqqR** [farmerbb/Taskbar#214]
I have this behavior when connecting to 4k screen:
![PXL_20210818_191434203 MP](https://user-images.githubusercontent.com/77554817/129959306-276cf7a7-af86-462f-827d-ae3c885c0407.jpg)

The space below taskbar is persistent. Even if I adjust taskbar to the right or left of screen, there is still a cutoff of the space at bottom of screen. It seems that this part of screen is deemed to be "unreachable" by taskbar.

Is this behavior expected?

**Comment by KevinBlue18** [comment]
Try to check if "C34e turn

**Comment by J4gQBqqR** [comment]
> Try to check if "C34e turn

Not sure what that means. Could you elaborate a little bit?

**Comment by farmerbb** [comment]
@J4gQBqqR Are you on Android 11?  Also do you have the navbar hidden?

**Comment by J4gQBqqR** [comment]
> @J4gQBqqR Are you on Android 11? Also do you have the navbar hidden?

Yes to both answers. But this blank bar behavior is not on 1080p screen, it only appears on 4k screen.

**Comment by farmerbb** [comment]
Ok, what's happening is that on Android 11, the app adds some white space below the Taskbar while in desktop mode, otherwise the Taskbar would actually draw behind the navbar instead.  The app doesn't check beforehand if the navbar is hidden, though.  I'll try to work around this scenario.
